app/javascript/packages/verify-flow/steps/password-confirm/personal-info-summary.tsx

```diff
@@ -0,0 +1,42 @@
+import { parse, format } from 'libphonenumber-js';
+import { t } from '@18f/identity-i18n';
+
+function PersonalInfoSummary({ pii }) {
+  const { firstName, lastName, dob, address1, address2, city, state, zipcode, ssn, phone } = pii;
+  const phoneNumber = parse(`+1${phone}`);
+  const formatted = format(phoneNumber, 'NATIONAL');
+
+  function getDateFormat(date) {
+    date = new Date(date);
+    const options = { year: 'numeric', month: 'long', day: 'numeric' };
+    return date.toLocaleDateString('en-US', options);
```

Comment:
What about non-English locales?

Reply:
what do you suggest?

Reply:
It should at least not show English labels, which is what it currently does. However, I just tested in the existing screen, and it appears we're showing English labels there currently as well (bug) 😕 

![image](https://user-images.githubusercontent.com/1779930/168809990-3dfcf29f-19d5-4a28-85cd-2affaaec079e.png)

One simple option could be to pull the language from the `<html>` tag:

```suggestion
    return date.toLocaleDateString(document.documentElement.lang, options);
```